pkgs/applications/video/epgstation/default.nix

```diff
@@ -49,21 +37,14 @@ let
   server = nodePackages.epgstation.override (drv: {
     inherit src;
 
-    bypassCache = false;
-
     # This is set to false to keep devDependencies at build time. Build time
     # dependencies are pruned afterwards.
     production = false;
 
     buildInputs = [ bash ];
-    nativeBuildInputs = [
-      nodejs
-      workaround-opencollective-buildfailures
+    nativeBuildInputs = drv.nativeBuildInputs ++ [
       makeWrapper
-    ] ++ (with nodePackages; [
-      node-pre-gyp
-      node-gyp-build
```

Comment:
These were seemingly not picked up by the build. Moving these to `pkgs/development/node-packages/default.nix` fixed the problem.